app/src/models.py

```diff
@@ -0,0 +1,49 @@
+from enum import Enum
+from sqlalchemy import Enum as SQLAlchemyEnum
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import CheckConstraint
+from datetime import datetime, timezone, timedelta
+from flask_bcrypt import Bcrypt
+
+bcrypt = Bcrypt()
+db = SQLAlchemy()
+
+# Predefined grade selections
+# Note: when adding a new user, use "grade=GradeEnum.NINTH" to assign a grade (in this example, 9th grade)
+class GradeEnum(Enum):
```

Comment:
Love the inclusion of specific grades, really helpful in terms of supporting profile information

May want to sync up with the other teams on the structure of this, (I.e is this a drop-down?)